SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -134,9 +103,9 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, simply add `-NetDriverOverrides=/Script/OnlineSubsystemUtils.IpNetDriver` to your command line arguments when you're launching the game (to do this in the Visual Studio Solution Explorer, right-click the project and select **Properties**, then **Debugging**, and add to the list of Command Arguments).
+    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor.
     >
-    > You can easily switch between networking modes using this argument. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > You can easily switch between networking modes using this tickbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
```

Comment:
I think we should say somewhere that if you want to build out your game, you have to turn spatial networking on/off by using the DefaultEngine.ini config

Reply:
Add similar in the above paragraph.